extract-data.c

```diff
@@ -0,0 +1,265 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <assert.h>
+#include <string.h>
+#include <sys/types.h>
+
+#define PTR(x) uint32_t
+
+#define PACKED __attribute__((packed))
+
+typedef struct {
+  uint8_t index; // absolute index to identify part?
+  uint8_t level; // level / quality?
+  uint8_t unk1; // ?
+  uint8_t category; // what kind of part it is?
+  int32_t price;
+  int32_t model; // model index
+  PTR(char*) title;
+} PACKED PartData;
+
+typedef struct {
+  uint32_t index;
+  uint32_t pod_model; // "Podd"
+  uint32_t pod_model_lod1; // "Malt": Good quality
+  uint32_t pod_model_lod2; // "Malt": Medium quality
+  uint32_t pod_model_lod3; // "Malt": Lowest quality
+  PTR(char*) first_name;
+  PTR(char*) last_name;
+  float unkf1; // Olganix: Scale for pod to fit into Cantina hologram ?
+  float unkf2; // Olganix: Scale for character to fit into Cantina hologram ?
+  uint32_t unk1;
+  uint32_t photo_sprite;
+  uint32_t flag_sprite;
+  uint32_t character_model;
+} PACKED PodracerData;
+
+typedef struct {
+  float AntiSkid; // 0
+  float TurnResponse; // 4
+  float MaxTurnRate; // 8
+  float Acceleration; // 12
+  float MaxSpeed; // 16
+  float AirbrakeInv; // 20
+  float DecelInv; // 24
+  float BoostThrust; // 28
+  float HeatRate; // 32
+  float CoolRate; // 36
+  float HoverHeight; // 40
+  float RepairRate; // 44
+  float BumpMass; // 48
+  float DmgImmunity; // 52
+  float IsectRadius; // 56 //FIXME: Not sure
+} PACKED PodracerHandlingData;
```

Comment:
`PodracerHandlingTableData`; this allows re-using this structure elsewhere.
See handling.h in future swe1r-re PR.